freebsd_sysctl/__version__.py

```diff
@@ -1,3 +1,56 @@
-VERSION = (0, 0, 8)
+#!/usr/bin/env python3
+import os
+import os.path
+import re
 
-__version__ = '.'.join(map(str, VERSION))
+# When a RELEASE file exists, the version is read from this file.
+# Otherwise the latest version from CHANGELOG.md is suffixed with -dev and
+# a git commit hash (if available).
```

Comment:
i don't understand why all this code needs to exist, and needs to exist in this project and be duplicated across projects, when it's only needed during a release